packages/textfield/package.json

```diff
@@ -22,6 +22,10 @@
     "lit-html": "^1.0.0",
     "tslib": "^1.10.0"
   },
+  "devDependencies": {
```

Comment:
I think these have to be dependencies, or else when a user compiles the textfield typings and don't also depend on these by some other means, then they'll get an unknown type error (since the textfield typings reference these typings)